reanahub/reana-workflow-controller
# jupyter: improve workspace and environment handling

Closes #305 

How to test:
1. Run a demo workflow
2. Open a new Jupyter Notebook
3. Check that the `REANA_WORKSPACE` environment variable is correctly set, both in notebooks and in terminals
4. Check that the workspace is now the CWD of the terminals
